bcmi/src/app/models/content/footer.ts

```diff
@@ -0,0 +1,19 @@
+export class Footer {
+  Navigate: FooterLink[];
+  Footer_bar: ExternalLink[];
+  About_title: string
+  About_description: string
+  Connect_title: string
+  Connect_description: string
+}
+
+
+class FooterLink{
+  Nav_heading: string
+  Footer_link: any[];
+}
+
+class ExternalLink{
```

Comment:
Would recommend putting this on one line (`class EternalLink { }`) if it's going to be empty.